app/components/Documentation/components/Pagination/PaginationDocumentation.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import { getIpfsHashFromBytes32 } from 'utils/ipfs';
+import createdHistory from 'createdHistory';
+import * as routes from 'routes-config';
+import useMediaQuery from 'hooks/useMediaQuery';
+import { getDataFromTree, getcurrentArrayIndex } from '../../helpers';
+import { DocumentationItemMenuType } from 'pages/Documentation/types';
+import ButtonPaginationDesktop from './ButtonPaginationDesktop';
+import ButtonPaginationMobile from './ButtonPaginationMobile';
+
+type PaginationDocumentationProps = {
+  documentationMenu: DocumentationItemMenuType;
+  id: string;
+  onClickPaginationArticleEditDocumentation?: () => void;
+};
+
+const PaginationDocumentation: React.FC<PaginationDocumentationProps> = ({
+  documentationMenu,
+  id,
+  onClickPaginationArticleEditDocumentation,
+}): JSX.Element => {
+  const isDesktop = useMediaQuery('(min-width: 577px)');
+  const treeArray = getDataFromTree(documentationMenu);
+
+  const currentArrayIndex = getcurrentArrayIndex(treeArray, id);
+
+  const isStartArticle = currentArrayIndex < 1;
+  const isLastArticle = currentArrayIndex === treeArray.length - 1;
+  const TITLE_LENGTH = isDesktop ? 26 : 50;
+  const TITLE_LAST_INDEX = isDesktop ? 25 : 49;
+  const TITLE_START_INDEX = 0;
```

Comment:
transfer to a separate file with constants and import

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1255/commits/303a47c06288eb7a7876b1135fe75d5894b3359f